Sources/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -14,12 +14,19 @@ class FunctionalTests_ItSpec: QuickSpec {
             expect(exampleMetadata!.example.name).to(equal(""))
         }
 
-        it("has a description with セレクター名に使えない文字が入っている 👊💥") {
-            let name = "has a description with セレクター名に使えない文字が入っている 👊💥"
+        it("has a description with セレクター名に使えない文字が入っている ��") {
+            let name = "has a description with セレクター名に使えない文字が入っている ��"
```

Comment:
Why are these emoji no longer displaying here? Could you revert this change, assuming you made one?